pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/patch/PatchFactory.java

```diff
@@ -29,6 +29,9 @@ public static BasePatch makePatch(Patch patch, NotificationUtil notificationUtil
             } else if (isParticipantIdForRecord(patch)) {
                 patcher = new ParticipantRecordPatch(patch);
             }
+        } else if (isParentParticipantId(patch) && !isMedicalRecordAbstractionFieldId(patch)
```

Comment:
@SimoneMaiwald does this boolean condition apply only for onc history detail patch? I am afraid that we may get into this if statement for other cases as well and we may face some exceptions

Reply:
all other patches should have already been caught by all the logic before

Reply:
all other patches at this point would throw a NullPatch. so either the patch works because it is a oncHistoryDetail field and the process -> update db will work because the column is the correct on or it will throw an error… but it would have thrown an NullPatch error next anyways…